contracts/GameJam.sol

```diff
@@ -1,10 +1,18 @@
 pragma solidity 0.5.11;
 
 contract GameJam {
+  uint public balance;
+
   mapping(address => string) public competitors;
 
   event CompetitorAdded(address competitor);
 
+  // Create a GameJam with a _balance
+  // which will be the amount to be distributed to winners
+  constructor(uint _balance) public {
```

Comment:
Rather than explicitly having a `_balance` in the constructor, should I be making use of the `payable` keyword?

Reply:
Na this is perfect, the `payable` keyword we use for passing eth to a function.

The idea is we have a management contract that instantiates this contract. And on the instantiation function, we will have the `payable` keyword

Reply:
Actually after investigating we need the `payable` function in the constructor.

like this:

```solidity
constructor (uint _balance)  payable public {
```

Then we can get the balance like this:

```solidity
address(this).balance
```

✊

Reply:
Added payable to constructor and now have a .sol test ✊